Foundation/NSAttributedString.swift

```diff
@@ -186,11 +186,11 @@ private extension AttributedString {
     }
 }
 
-extension AttributedString: _CFBridgable {
+extension NSAttributedString: _CFBridgable {
```

Comment:
In early days I used Apple's own Swift book as a reference. So far the use of `:` seems to be just like in natural English: no space before `:`, one space after. Same as `,`. The only exception being ternary operator `a ? b : c`. Latest documentation is still following this style, e.g. https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html#//apple_ref/doc/uid/TP40014097-CH12-ID145

But then Swift was open sourced and I noticed that there's not much consistency. So these days I wonder what approach the core team and open source community takes. I guess there's tons of really important stuff to do :)

We personally were early adopters of SwiftLint and so far it does a great job for us. I'm curious if open source Swift code will use third party tools or if Apple is planning to come up with this functionality built-in in Xcode.